proposals/market_perp_btcusd.go

```diff
@@ -0,0 +1,275 @@
+package proposals
+
+import (
+	"fmt"
+	"time"
+
+	"code.vegaprotocol.io/vega/libs/ptr"
+	"code.vegaprotocol.io/vega/protos/vega"
+	commandspb "code.vegaprotocol.io/vega/protos/vega/commands/v1"
+	datav1 "code.vegaprotocol.io/vega/protos/vega/data/v1"
+	"github.com/vegaprotocol/devopstools/tools"
+)
+
+func NewBTCUSDPerpetualMarketProposal(
+	settlementVegaAssetId string,
+	decimalPlaces uint64,
+	oraclePubKey string,
+	closingTime time.Time,
+	enactmentTime time.Time,
+	extraMetadata []string,
+) *commandspb.ProposalSubmission {
+	var (
+		reference = tools.RandAlpaNumericString(40)
+		Name      = fmt.Sprintf("BTCUSD Perpetual")
+	)
+
+	contractABI := "{\"inputs\":[],\"name\":\"latestAnswer\",\"outputs\":[{\"internalType\":\"int256\",\"name\":\"\",\"type\":\"int256\"}],\"stateMutability\":\"view\",\"type\":\"function\"}"
+
+	return &commandspb.ProposalSubmission{
+		Reference: reference,
+		Rationale: &vega.ProposalRationale{
+			Title:       "New BTCUSD perpetual market",
+			Description: "New BTCUSD perpetual Market",
+		},
+		Terms: &vega.ProposalTerms{
+			ClosingTimestamp:   closingTime.Unix(),
+			EnactmentTimestamp: enactmentTime.Unix(),
+			Change: &vega.ProposalTerms_NewMarket{
+				NewMarket: &vega.NewMarket{
+					Changes: &vega.NewMarketConfiguration{
+						DecimalPlaces:           decimalPlaces,
+						PositionDecimalPlaces:   3,
+						LinearSlippageFactor:    "0.1",
+						QuadraticSlippageFactor: "0.1",
+						Instrument: &vega.InstrumentConfiguration{
+							Name: Name,
+							Code: "BTCUSD.MF21",
+							Product: &vega.InstrumentConfiguration_Perpetual{
+								Perpetual: &vega.PerpetualProduct{
+									SettlementAsset: settlementVegaAssetId,
+									QuoteName:       "USD",
+									DataSourceSpecForSettlementData: &vega.DataSourceDefinition{
+										SourceType: &vega.DataSourceDefinition_External{
+											External: &vega.DataSourceDefinitionExternal{
+												SourceType: &vega.DataSourceDefinitionExternal_EthOracle{
+													EthOracle: &vega.EthCallSpec{
+														Address: "0x5fb1616F78dA7aFC9FF79e0371741a747D2a7F22", // chainlin BTC/USD
```

Comment:
I made a mistake here

```suggestion
														Address: "0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43", // chainlin BTC/USD
```